src/components/CreatePostForm.tsx

```diff
@@ -0,0 +1,65 @@
+// Render Prop
+import React from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { PostItemInterface } from '../interfaces/Post';
+
+
+const CreatePostForm = ({
+  onSavePost
+}: { onSavePost: (i: PostItemInterface) => void}) => (
+  <div>
+    <h1>New Post</h1>
```

Comment:
From a semantic point of view, why would you go with an `<h1 />` here?

Reply:
just to have the title, i did not spend time the design